SWLOR.Game.Server/ChatCommand/PlayVFX.cs

```diff
@@ -0,0 +1,44 @@
+﻿using SWLOR.Game.Server.ChatCommand.Contracts;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.GameObject;
+using System;
+using SWLOR.Game.Server.NWN;
+using SWLOR.Game.Server.NWN.Enum;
+using SWLOR.Game.Server.NWN.Enum.VisualEffect;
+using static SWLOR.Game.Server.NWN._;
+using SWLOR.Game.Server.Service;
+
+namespace SWLOR.Game.Server.ChatCommand
+{
+    [CommandDetails("Switches the active language. Use /language help for more information.", CommandPermissionType.DM | CommandPermissionType.Admin)]
```

Comment:
Merged this already but just noticed the description is wrong. Can you fix that? @givemedeath